core/src/main/java/com/datastax/dse/driver/internal/core/graph/binary/AbstractSimpleGraphBinaryCustomSerializer.java

```diff
@@ -105,11 +103,11 @@ public T readValue(Buffer buffer, GraphBinaryReader context, boolean nullable)
     final int valueLength = buffer.readInt();
 
     if (valueLength <= 0) {
-      throw new SerializationException(String.format("Unexpected value length: %d", valueLength));
+      throw new IOException(String.format("Unexpected value length: %d", valueLength));
```

Comment:
I think this is safe. I explored the call hierarchy of these methods, and no caller catches `SerializationException` nor applies any custom processing to instances of `SerializationException`. Actually the whole Tinkerpop project is moving away from `SerializationException`  it seems: I could only find 2 `catch (SerializationException e)` blocks in the whole codebase. See also [TINKERPOP-2179](https://issues.apache.org/jira/browse/TINKERPOP-2179) for context.